Integrations/LogRhythmRest/LogRhythmRest.py

```diff
@@ -556,6 +556,8 @@ def get_alarm_events(data_args):
     events = []
 
     for block in res:
+        if not block['DrillDownLogs']:
```

Comment:
better to be on the safe-side
```suggestion
        if not block.get('DrillDownLogs'):
```